OCR/frontend/src/pages/ExtractProcess.tsx

```diff
@@ -60,9 +68,7 @@ const ExtractProcess = () => {
         file_image: templates[0].pages[0].sourceImage,
         results,
       }
-        
-      
-      
+              
       localStorage.setItem("extractedData", JSON.stringify(transformedResponses));
```

Comment:
is this the correct place to save the returned data? I thought the review page was checking `submission` key